File: RootTools2/src/main/java/com/rarnu/tools/neo/data/BuildPropInfo.java
package com.rarnu.tools.neo.data;

import java.io.Serializable;

public class BuildPropInfo implements Serializable {

    public String buildName;
    public String buildValue;

    public static BuildPropInfo parse(String str) {
        BuildPropInfo info = new BuildPropInfo();
        info.buildName = str.substring(0, str.indexOf("=")).trim();
        info.buildValue = str.substring(str.indexOf("=") + 1).trim();
        return info;
    }
}

File: RootTools2/src/main/kotlin/com/rarnu/tools/neo/data/BuildPropInfo.kt
package com.rarnu.tools.neo.data

import com.rarnu.common.toPair
import java.io.Serializable

data class BuildPropInfo(var buildName: String, var buildValue: String) : Serializable {
    companion object {
        fun parse(str: String): BuildPropInfo = str.toPair().let { BuildPropInfo(it.first, it.second) }
    }
}
